src/backgroundblurprocessor/BackgroundBlurProcessorProvided.ts

```diff
@@ -139,15 +146,27 @@ export default class BackgroundBlurProcessorProvided implements BackgroundBlurPr
 
     this.spec = spec;
     this.logger = options.logger;
+    this.videoFramesPerFilterUpdate = 1;
 
     this.setBlurStrength(options.blurStrength);
     this.delegate = new BackgroundBlurVideoFrameProcessorDelegate();
     this.frameCounter = new BackgroundFilterFrameCounter(
       this.delegate,
       options.reportingPeriodMillis,
+      options.filterCPUUtilization,
       this.logger
     );
 
+    this.cpuMonitor = new BackgroundFilterMonitor(options.reportingPeriodMillis * 5, {
+      reduceCPUUtilization: () => {
+        this.updateVideoFramesPerFilterUpdate(Math.min(this.videoFramesPerFilterUpdate + 1, 5));
```

Comment:
nit: Can we create constant for these value? Not sure what 5 or 1 mean

Reply:
will do.